### DISCUSSION TITLE
bug in the zotero note editor

### DISCUSSION THREAD

**Discussion by Brilliancy909** [retorquere/zotero-better-bibtex#2751] [Q&A] [answered]
在添加公式以后无法直接进行中文的输入，英文可以直接的输入，请问这个问题大家是怎么解决的

**Comment by retorquere** [comment]
English please.

**Reply by Brilliancy909** [reply]
After adding the formula, it is impossible to input Chinese directly, but English can be input directly. How do you solve this problem?

**Reply by retorquere** [reply]
To input Chinese where?

**Reply by Brilliancy909** [reply]
After formula editing, the formula editing bar pops up. eg  ( $ \theta $    You can use English directly after this, but you can't use Chinese  )

**Reply by retorquere** [reply]
Can you make a screen recording? I don't understand what you see happening. I don't know what a "formula editing bar" is, and nothing is supposed to pop up.

**Reply by retorquere** [reply]
BTW if it really is a bug you will have to open an issue. But let's first see what is going on.

**Reply by Brilliancy909** [reply]
https://github.com/retorquere/zotero-better-bibtex/assets/129017073/fd7018bd-ac27-4b8a-bccc-5cb51b6085d7

**Reply by Brilliancy909** [reply]
I don't know if you can see the problem. After inputting English, I switched the Chinese input method, and then found that the words input by the Chinese input method were not displayed, and then switched back to the English input method

**Reply by retorquere** [reply]
The video doesn't show anything for me, it's just blank 29 seconds.

**Reply by retorquere** [reply]
Oh I see it now in Safari. But this looks like a note field? It doesn't look at all like the citation key formula editor. Are you in the preferences screen? Can you include the preference screen in your recording?

**Reply by Brilliancy909** [reply]
http://1258511654.vod2.myqcloud.com/1ff9b668vodcq1258511654/bf09ca39387702307278929433/J2ADOhKZjNoA.mp4

**Reply by retorquere** [reply, marked as answer]
Yeah that's the Zotero note editor, that isn't related to this plugin at all. If there's a problem with that you'll have to report it to the zotero developers on their forum.

**Reply by Brilliancy909** [reply]
oh,thank you!
